### ISSUE TITLE
Control Statement Parentheses Violation not detecting all violations

### ISSUE THREAD

**Issue by daniel-beard** [realm/SwiftLint#187]
swiftlint version 0.3.0

Reproducible case:

```
import Foundation

class TestClass: TestObject {
    
    func enterUserFirstLastName(firstName: String,lastName: String) {
        let clearTextButton = app.buttons[kClearText]
        let firstNameField = app.tables.textFields[kFirstName]
        let lastNameField = app.tables.textFields[kLastName]
        firstNameField.tap()
        if(clearTextButton.exists) {
            clearTextButton.tap()
        }
        firstNameField.typeText(firstName)
        lastNameField.tap()
        if(clearTextButton.exists) {
            clearTextButton.tap()
        }
        lastNameField.typeText(lastName)
    }
}
```

Swiftlint output:

```
Linting Swift files in current working directory
Linting 'test.swift' (1/1)
/Users/dbeard/iphone/test.swift:1: warning: Leading Whitespace Violation: File shouldn't start with whitespace: currently starts with 1 whitespace characters
/Users/dbeard/iphone/test.swift:5: warning: Trailing Whitespace Violation: Line #5 should have no trailing whitespace
/Users/dbeard/iphone/test.swift:11:9: warning: Control Statement Parentheses Violation: if statements shouldn't wrap their conditionals in parentheses.
Done linting! Found 3 violations, 0 serious in 1 file.
```

Swiftlint picks up the first violation, but does not detect the second control statement violation.
